pages/deploy-service-broker-locally.md

```diff
@@ -0,0 +1,195 @@
+uaac target https://192.168.50.6:8443
+uaac token client get uaa_admin -s ((uaa_admin_client_secret))
+```
+The `uaa_admin_client_secret` can be found in `creds.yml`. Create a new client and a user like this:
+```
+uaac client add webappclient -s webappclientsecret \ 
+--name WebAppClient \ 
+--scope resource.read,resource.write,openid,profile,email,address,phone \ 
+--authorized_grant_types authorization_code,refresh_token,client_credentials,password \ 
+--authorities uaa.resource \ 
+--redirect_uri http://localhost:8080/custom/v2/authentication/
+
+uaac user add appuser -p appusersecret --emails appuser@acme.com
+	
+uaac group add resource.read
+uaac group add resource.write
+
+uaac member add resource.read appuser
+uaac member add resource.write appuser
+
+```
+For more information on uaac read [here](https://docs.pivotal.io/platform/application-service/2-8/uaa/uaa-user-management.html).
+
+The bosh lite uaa uses the same IP as the director, and the port `8443`. The service broker configuration YAML needs to be configured like this:
+
+```yaml
+spring:
+  security:
+      oauth2:
+        resourceserver:
+          jwt:
+            issuer-uri: "https://192.168.50.6:8443/oauth/token"
+catalog:
+  services:
+      dashboard:
+        auth_endpoint: "https://192.168.50.6:8443/oauth"
+        url: "http://localhost:8080/custom/v2/authentication"
+      dashboard_client:
+        id: webappclient
+        redirect_uri: "http://localhost:8080/custom/v2/authentication/"
+        secret: webappclientsecret
+```
+
+To access UAA the service broker needs to trust the SSL certificate that protects Uaa. There are two options, trust all self-signed certificates or add the uaa ca from the `creds.yml` to the service broker. To trust all certificates configure this:
+
+```yaml
+spring:
+  ssl:
+    acceptselfsigned: true
+```
+
+Add the director ssl ca from the `creds.yml` like below, if you don't want to just accept everything:
+
+```yaml
+spring:
+  ssl:
+    certificates:
+      uaa_ssl: |
+        -----BEGIN CERTIFICATE-----
+        .....
+        -----END CERTIFICATE-----
+```
+The ca can be found in `creds.yml` under the key `director_ssl.ca`. After this the dashboard of a deployed service is accessible through its dashboard URL. The user `appuser` with password `appusersecret` can be used to log in.
+
+### Connect to bosh-lite credhub
+To use the bosh lite credhub as a credentials store, the service broker has to authenticate itself with the uaa. Credhub runs on the same IP as the director and uses the port `8844` The necessary `client-id` is called `credhub-admin` the `client-secret` is saved in the `creds.yml` under `credhub_admin_client_secret`. Credhub uses a different certificate that the uaa, so it all self-signed certificates need to be accepted or the `credhub ca` needs to be accepted in addition to the `director ca`. The `credhub ca` is saved in the `creds.yml` file in `credhub_ca.ca`. A credhub configuration with bosh lite looks like this:
```

Comment:
Credhub uses a different certificate thaN the uaa,

Reply:
fixed